draft-ietf-quic-multipath.md

```diff
@@ -625,6 +630,41 @@ delay will most of the time relate to the path with the shortest latency.
 To collect ACK delays on all the paths, hosts should rely on time stamps
 as described in {{QUIC-Timestamp}}.
 
+### ECN and NULL CID Considerations
+
+ECN feedback in QUIC is provided based on counters in the ACK frame
+(see {{Section  19.3.2. of QUIC-TRANSPORT}}). These counters are specific
+to a number space. When sending to NULL CID receivers, teh same number space is
+used for multiple paths, and the ECN feedback cannot unambiguously be assigned to
+a path.
+
+If the sender marks its packets with the ECN capable flags, the network
+will expect standard reactions to ECN marks, such as slowing down
+transmission on the sending path. As the sending path is ambiguous,
+the sender must treat a CE marking as a congestion signal on all sending
+paths.
+
+A host that is sending over multiple paths to a NULL CID receiver
+MAY disable ECN marking and
+send all subsequent packets as Not-ECN capable.
+
+### Restricted Sending to NULL CID Peer
+
+Hosts that are designed to support multipath using multiple number spaces
+MAY adopt a conservative posture after negotiating multipath support with
+a peer using NULL CID. The simplest posture is to elect to only send
+data on one path at a time, while accepting packets on all acceptable
+paths. In that case:
+
+* the attribution of packets to path discussed in {{null-cid-loss-and-congestion}}
+  are easy to solve because packets are sent on a single path,
+* the ACK Delays are mostly correct,
```

Comment:
why "mostly"?

Reply:
:-)
need to find another way of saying that without a number space per path this is harder to get right.

Reply:
Removed "mostly"